Maple2.Database/Model/Character.cs

```diff
@@ -101,6 +102,7 @@ internal class Character {
             StorageCooldown = other.Cooldown.Storage,
             DeleteTime = other.DeleteTime.ToEpochSeconds(),
             Channel = other.Channel,
+            Achievements = new Dictionary<int, Maple2.Model.Game.Achievement>(),
```

Comment:
The reason is because it avoids doubling up fetching achievements twice.
I need to get the character's trophy count for `CharacterListPacket` when the user has logged in.
I would need to get it again when the player logs into the player.
I fetch it once this way.

Reply:
You would have to fetch twice anyway right? Once on `LoginServer` and once on `GameServer`.

I think a potential solution to this is to just store the Trophy counts for the character (used for display only, not as source of truth), but once you are logged in it will use values from `AchievementManager`.

Alternatively, it may be possible to craft a SQL query that can return trophy counts without needing to return all the achievement rows.

something like:
```sql
Count = SELECT SUM(GradeCount) FROM achievements WHERE OwnerId=AccountId OR OwnerId=CharacterId;
```

note: you would probably want to store a value for each row equivalent to `Grades.Count`